src/main/scala/org/constellation/infrastructure/endpoints/TransactionEndpoints.scala

```diff
@@ -68,12 +68,19 @@ class TransactionEndpoints[F[_]](implicit F: Concurrent[F]) extends Http4sDsl[F]
 
   def peerEndpoints(transactionService: TransactionService[F], metrics: Metrics) =
     getTransactionEndpoint(transactionService) <+>
-      getBatchEndpoint(metrics, transactionService)
+      getBatchEndpoint(metrics, transactionService) <+>
+      getLastTransactionEndpoint(transactionService)
 
   private def getTransactionEndpoint(transactionService: TransactionService[F]): HttpRoutes[F] = HttpRoutes.of[F] {
     case GET -> Root / "transaction" / hash =>
       transactionService.lookup(hash).map(_.asJson).flatMap(Ok(_))
   }
+
+  /** Gets the reference to the last transaction for a given address or an empty reference if no transactions yet. */
+  private def getLastTransactionEndpoint(transactionService: TransactionService[F]): HttpRoutes[F] = HttpRoutes.of[F] {
```

Comment:
It's not a last transaction but last transaction reference. The comment is also wrong, because you've mixed there the implementation part of the service and it's just a call for this service. Just a proper name of the endpoint would be fine, you don't need to comment everything in the code.